apps/frontend/src/pages/project/project.tsx

```diff
@@ -37,6 +37,14 @@ const Project = (): JSX.Element => {
 
 	return (
 		<PageLayout isLoading={isLoading}>
+			<div className={styles["breadcrumb-container"]}>
+				{project && (
```

Comment:
should be a boolean value
```suggestion
				{hasProject && (
```